source/features/prevent-link-loss.tsx

```diff
@@ -7,27 +7,43 @@ import * as pageDetect from 'github-url-detection';
 import * as textFieldEdit from 'text-field-edit';
 
 import features from '.';
-import {prCommitUrlRegex, preventPrCommitLinkLoss, prCompareUrlRegex, preventPrCompareLinkLoss} from '../github-helpers';
+import {
+	prCommitUrlRegex, preventPrCommitLinkLoss,
+	prCompareUrlRegex, preventPrCompareLinkLoss,
+	discussionUrlRegex, preventDiscussionLinkLoss
+} from '../github-helpers';
 
 function handleButtonClick({delegateTarget: fixButton}: delegate.Event<MouseEvent, HTMLButtonElement>): void {
 	const field = fixButton.form!.querySelector('textarea')!;
 	textFieldEdit.replace(field, prCommitUrlRegex, preventPrCommitLinkLoss);
 	textFieldEdit.replace(field, prCompareUrlRegex, preventPrCompareLinkLoss);
+	textFieldEdit.replace(field, discussionUrlRegex, preventDiscussionLinkLoss);
 	fixButton.parentElement!.remove();
 }
 
+function getRghIssueLink(issueShortcut: string): Element {
+	const issueNumber = issueShortcut.slice(1);
+	return <a target="_blank" rel="noopener noreferrer" href={`https://github.com/sindresorhus/refined-github/issues/${issueNumber}`}>{issueShortcut}</a>;
+}
+
 function getUI(field: HTMLTextAreaElement): HTMLElement {
-	return select('.rgh-fix-pr-commit-links-container', field.form!) ?? (
-		<div className="flash flash-warn mb-2 rgh-fix-pr-commit-links-container">
-			<AlertIcon/> Your PR Commit link may be <a target="_blank" rel="noopener noreferrer" href="https://github.com/sindresorhus/refined-github/issues/2327">misinterpreted by GitHub.</a>
-			<button type="button" className="btn btn-sm primary flash-action rgh-fix-pr-commit-links">Fix link</button>
+	return select('.rgh-prevent-link-loss-container', field.form!) ?? (
+		<div className="flash flash-warn mb-2 rgh-prevent-link-loss-container">
+			<AlertIcon/> Your link may be misinterpreted by GitHub (see {getRghIssueLink('#2327')}, {getRghIssueLink('#3528')}, {getRghIssueLink('#4357')}).
```

Comment:
Instead of dropping it in the function, just pass a number 
```suggestion
			<AlertIcon/> Your link may be misinterpreted by GitHub (see {getRghIssueLink(2327)}, {getRghIssueLink(3528)}, {getRghIssueLink(4357)}).
```

Reply:
I was hoping that `'#2327'` would be linkified by RGH when reading the file's source code. But since it does not (only comments, right?), I'll simplify it.